README.md

```diff
@@ -37,6 +37,14 @@ By default Docker images built using clearwater-docker will pull the latest stab
 
 ![alt text](docs/images/clearwater-docker_in_scope.jpg "Clearwater-Docker dispayed in Scope")
 
+## Limitations
+
+Note that scaling of Docker deployments is a work in progress and there are currently a number of known issues...
+
+* Homestead-prov and Ellis don’t load balance across multiple Cassandra nodes.
```

Comment:
This is sad :( 
Is it because of the way python creates the connections that it only happens at start of day or something?